usage/src/main/java/com/cloud/usage/parser/BackupUsageParser.java

```diff
@@ -100,6 +89,32 @@ public static boolean parse(AccountVO account, Date startDate, Date endDate) {
         return true;
     }
 
+    protected static final Map<String, BackupInfo> getVmUsageMap(final List<UsageBackupVO> usageBackups) {
+        final Map<Long, Long> vmOffering = new HashMap<Long, Long>();
+        final Map<Long, Integer> vmOfferingIndex = new HashMap<Long, Integer>();
+        final Map<String, BackupInfo> vmUsageMap = new HashMap<>();
+        for (final UsageBackupVO usageBackup : usageBackups) {
+            final Long vmId = usageBackup.getVmId();
+            final Long zoneId = usageBackup.getZoneId();
+            final Long offeringId = usageBackup.getBackupOfferingId();
+            Integer index = vmOfferingIndex.get(vmId) == null ? -1 : vmOfferingIndex.get(vmId);
+            boolean isVmOfferingEqualsToPrevious = vmOffering.get(vmId) != null && vmOffering.get(vmId) == offeringId;
```

Comment:
```suggestion
            boolean isVmOfferingEqualsToPrevious = offeringId.equals(vmOffering.get(vmId));
```

Reply:
maybe also change the name to `isVmOfferingEqualToPrevious`, without the 3rd person singular 's'. Also the name `vmOfferingEqualsToPrevious` is better.

Reply:
@DaanHoogland thanks. done!